test/utils.js

```diff
@@ -0,0 +1,20 @@
+import { parse } from 'graphql';
+import { validate } from 'graphql/validation';
+import { buildASTSchema } from 'graphql/utilities/buildASTSchema';
+
+export function validateSchemaString(schemaString, rules) {
```

Comment:
This is a good start.

In the future, I kinda want to do like `graphql-js` and have `assertPassesRule` and `assertFailsRule` similar to this: https://github.com/graphql/graphql-js/blob/0eb070fed365095666847d5402e759ebe5854e64/src/validation/__tests__/ArgumentsOfCorrectType-test.js#L183